docs/reference/scripting.asciidoc

```diff
@@ -4,81 +4,59 @@
 [partintro]
 --
 With scripting, you can evaluate custom expressions in {es}. For example, you
-could use a script to return "script fields" as part of a search request or
-evaluate a custom score for a query.
+can use a script to return a computed value as a field or evaluate a custom
+score for a query.
 
-The default scripting language is <<modules-scripting-painless, `Painless`>>.
-Additional `lang` plugins enable you to run scripts written in other languages.
-Everywhere a script can be used, you can include a `lang` parameter
-to specify the language of the script.
+The default scripting language is <<modules-scripting-painless,Painless>>.
+Additional `lang` plugins are available to run scripts written in other
+languages. You can specify the language of the script anywhere that scripts run.
 
 [discrete]
-== General-purpose languages
+[[scripting-available-languages]]
+== Available scripting languages
 
-These languages can be used for any purpose in the scripting APIs,
-and give the most flexibility.
-
-[cols="<,<,<",options="header",]
-|=======================================================================
-|Language
-    |Sandboxed
-    |Required plugin
-
-|<<modules-scripting-painless, `painless`>>
-    |yes
-    |built-in
-
-|=======================================================================
-
-[discrete]
-== Special-purpose languages
-
-These languages are less flexible, but typically have higher performance for
-certain tasks.
+Painless is purpose-built for {es}, can be used for any purpose in the
+scripting APIs, and provides the most flexibility. The other languages are less
```

Comment:
With the second sentence, I'm worried about implying that it may be worth using a different language for better performance. This use case is so rare and advanced imo, that I think we may want to re-word this in a way to not include performance as a reason for an alternative language.

Reply:
Per our chat, let's change this to read:
>The other languages are less flexible, but can be useful for specific purposes.